test-apps/testpurchasesandroidcompatibility/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 
 android {
     namespace = "com.revenuecat.testpurchasesandroidcompatibility"
-    compileSdk = 33 // Keeping it at 33 to test compatibility with purchases-android
+    compileSdk = 34 // Keeping it at 34 to test compatibility with purchases-android
```

Comment:
Hmm yeah... From what I remember, we added this because we added a dependency that made is incompatible with apps targeting the same SDK version as us... But I think it should be fine to test against 34 now 👍